.changeset/loud-walls-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'next-urql': major
```

Comment:
```suggestion
'next-urql': patch
```

Controversial opinion, but I'd choose a `patch` here. We're technically not adding a new feature, we're just exposing it as it was supposed to

Reply:
Oooh, even better!